packages/unlock-js/src/__tests__/integration/walletServiceIntegration.test.js

```diff
@@ -41,15 +37,24 @@ const versions = {
   // 'v10' : ['v9', 'v10', etc]
 }
 
-describe.each(Object.keys(versions))('%s', (versionName) => {
+describe.each(Object.keys(versions))('Unlock %s', (versionName) => {
   let walletService
   let web3Service
+  let ERC20Address
 
   beforeAll(async () => {
+    // deploy ERC20 and set balances
+    ERC20Address = await nodeSetup()
+
+    const [signer] = await ethers.getSigners()
+    const ethersProvider = signer.provider
+
+    // pass hardhat ethers provider
+    networks[chainId].ethersProvider = ethersProvider
```

Comment:
this requires a change in unlock-js itself: the `networks` object pass to constructor now accept a `ethersProvider` key that contains an ethers js provider.